spec/requests/api/reports_spec.rb

```diff
@@ -117,7 +161,7 @@
 
       expect do
         api_basic_authorize action_identifier(:reports, :run)
-        run_post "#{reports_url(report.id)}", :action => "run"
+        run_post reports_url(report.id).to_s, :action => "run"
```

Comment:
Mmh this is the suggestion of MiqBot because of rubocop prefere "to_s" rather than string interpolation. But it's right, it's the same, string interpolation call to_s and is better readable than to_s.